src/templates/QuickstartDetails.js

```diff
@@ -75,7 +75,12 @@ const QuickstartDetails = ({ data, location }) => {
 
   return (
     <>
-      <DevSiteSeo title={pack.name} location={location} />
+      <DevSiteSeo
+        title={pack.name}
+        type="quickstarts"
+        location={location}
+        quickStartName={pack.title}
```

Comment:
It looks like the `<DevSiteSeo>` component [accepts a `meta` prop](https://github.com/newrelic/developer-website/blob/a9b39ff31eb7add2205f28d9dc680dfd596ff2cd/src/components/DevSiteSeo.js#L121) for supplying custom meta tags as needed. Perhaps we should take advantage of that, rather than extending the component for just this one use-case.

If we did that, we'd build the meta tags for the quickstart in this file and just pass them in like so:

```jsx
const quickstartMeta = [
  {
    name: 'quick_start_name',
    class: 'swiftype',
    'data-type': 'string',
    content: pack.title,
  },
  {
    name: 'type',
    class: 'swiftype',
    'data-type': 'enum',
    content: 'quickstarts',
  },
];

// ...

<DevSiteSeo
  title={pack.title}
  type="quickstarts"
  location={location}
  meta={quickstartMeta}
/>
```

Reply:
works for me! 🎉